activerecord/lib/active_record/encryption/encrypted_attribute_type.rb

```diff
@@ -59,7 +59,7 @@ def previous_types # :nodoc:
       end
 
       def support_unencrypted_data?
-        ActiveRecord::Encryption.config.support_unencrypted_data && scheme.support_unencrypted_data? && !previous_type?
+        scheme.support_unencrypted_data? && !previous_type?
```

Comment:
I _think_ this makes sense because `scheme.support_unencrypted_data?` checks the value of config _after_ the scheme attribute:

https://github.com/rails/rails/blob/6b5c91d2af47c976b57d8a90d372b8ec8fb7c7bd/activerecord/lib/active_record/encryption/scheme.rb#L49

Reply:
Yeah exactly.